File: app/src/main/java/com/blues/module_kaiyan/search/OEHotSearchViewHolder.java
package com.blues.module_kaiyan.search;

import android.view.ViewGroup;
import android.widget.TextView;

import com.blues.R;
import com.jude.easyrecyclerview.adapter.BaseViewHolder;

/**
 * User : Blues
 * Date : 2019/8/28
 * Time : 18:30
 */

public class OEHotSearchViewHolder extends BaseViewHolder<String> {

    private TextView tvHotSearch;

    public OEHotSearchViewHolder(ViewGroup parent) {
        super(parent, R.layout.item_hot_search);

        tvHotSearch = $(R.id.tv_hot_search);
    }

    @Override
    public void setData(String data) {
        tvHotSearch.setText(data);
    }
}

File: app/src/main/java/com/blues/kaiyan/list/view/OEHotSearchViewHolder.kt
package com.blues.kaiyan.list.view

import android.view.ViewGroup
import com.blues.R
import android.widget.TextView
import com.jude.easyrecyclerview.adapter.BaseViewHolder

/**
* File: com.blues.kaiyan.list.view.OEHotSearchViewHolder.kt
* Description: xxx
*
* @author: lanhuajian
* @time: 2021/7/30
**/

class OEHotSearchViewHolder(parent: ViewGroup?) :
    BaseViewHolder<String?>(parent, R.layout.item_hot_search) {

    private val tvHotSearch: TextView = itemView.findViewById(R.id.tv_hot_search)

    override fun setData(data: String?) {
        tvHotSearch.text = data
    }
}